### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mchv**
## What does this change?

Update dependencies to test quickly how many vulnerabilities are addressed doing so.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -3,18 +3,16 @@ import com.gu.riffraff.artifact.RiffRaffArtifact.autoImport.{riffRaffArtifactRes
 val compilerOptions = Seq(
   "-deprecation",
   "-Xfatal-warnings",
-  "-encoding", "UTF-8",
-  "-target:jvm-1.8"
```
Comment (mchv) [inline build.sbt:7]: As mentioned [on documentation](https://docs.scala-lang.org/overviews/compiler-options/index.html) `target` has been deprecated and replaced with `release`, however:
- `8` is the default for `target`
- We have moved to Java 11 as our default

**Review by kenoir — APPROVED** [review]

**Comment by kenoir** [comment]
Snyk report pre-merge:

Screenshot 2022-11-10 at 09 19 11

**Comment by kenoir** [comment]
Snyk report post-merge:

Screenshot 2022-11-10 at 09 28 03

**Comment by kenoir** [comment]
> Update dependencies to test quickly how many vulnerabilities are addressed doing so.

So i'd say _most_ in this case?

@mchv

**Comment by mchv** [comment]
Thanks @kenoir! It took me a short period of time to do it manually, so that reinforce my thinking we should make sure teams update their dependencies in the timeline we discussed. 

Please note however that I have not released a new version of client.

**Pull request merged**